include/swift/SIL/SILFunction.h

```diff
@@ -175,7 +175,7 @@ class SILDifferentiableAttr final {
   SILFunction *getOriginal() const { return Original; }
 
   const SILAutoDiffIndices &getIndices() const { return indices; }
-  void setIndices(const SILAutoDiffIndices &indices) {
+  void setIndices(SILAutoDiffIndices indices) {
```

Comment:
This change is not useful -- what's it for?

Reply:
The new usage of `setIndices` doesn't compile unless `const ... &` is removed.

Reply:
You just need to define a new variable first.
```cpp
SILAutoDiffIndices newIndices(...);
attr->setIndices(newIndices);
```

This is not a pointer-sized struct, so passing it by value will incur two copies for other callers.

Reply:
My bad, `void setIndices(const SILAutoDiffIndices &indices)` is actually valid. Reverted in https://github.com/apple/swift/pull/26202/commits/91bf05de2efeac98be6a6771ee92c467ed2b390f.